client/package.json

```diff
@@ -11,42 +11,54 @@
     "build:prod": "ng build --baseHref=/nfe/ --prod=true --aot=true --buildOptimizer=true --sourceMap=false --statsJson=true --optimization=true --progress=true",
     "test": "jest",
     "test:watch": "jest --watch",
-    "lint": "ng lint && sass-lint --verbose --no-exit",
+    "lint": "ng lint && yarn lint:sass",
+    "lint:fix": "ng lint --fix && lint:sass",
+    "lint:sass": "sass-lint --verbose --no-exit",
     "format:fix:staged": "pretty-quick --staged --verbose --pattern \"**/*.{js,json,md,ts}\""
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~10.2.3",
+    "@angular/animations": "~11.0.0",
     "@angular/cdk": "^10.2.7",
-    "@angular/common": "~10.2.3",
-    "@angular/compiler": "~10.2.3",
-    "@angular/core": "~10.2.3",
-    "@angular/forms": "~10.2.3",
+    "@angular/common": "~11.0.0",
+    "@angular/compiler": "~11.0.0",
+    "@angular/core": "~11.0.0",
+    "@angular/forms": "~11.0.0",
     "@angular/material": "^10.2.7",
-    "@angular/platform-browser": "~10.2.3",
-    "@angular/platform-browser-dynamic": "~10.2.3",
-    "@angular/router": "~10.2.3",
+    "@angular/platform-browser": "~11.0.0",
+    "@angular/platform-browser-dynamic": "~11.0.0",
+    "@angular/router": "~11.0.0",
     "@briebug/jest-schematic": "^2.1.1",
     "@schuchard/prettier": "^3.1.0",
     "rxjs": "~6.5.4",
     "tslib": "^2.0.0",
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1002.0",
-    "@angular/cli": "~10.2.0",
-    "@angular/compiler-cli": "~10.2.3",
-    "@angular/language-service": "~10.2.3",
+    "@angular-devkit/build-angular": "~0.1100.0",
+    "@angular-eslint/builder": "0.6.0-beta.0",
+    "@angular-eslint/eslint-plugin": "0.6.0-beta.0",
+    "@angular-eslint/eslint-plugin-template": "0.6.0-beta.0",
+    "@angular-eslint/schematics": "^0.6.0-beta.0",
+    "@angular-eslint/template-parser": "0.6.0-beta.0",
+    "@angular/cli": "~11.0.0",
+    "@angular/compiler-cli": "~11.0.0",
+    "@angular/language-service": "~11.0.0",
     "@ngneat/spectator": "^5.3.1",
     "@types/jest": "25.1.4",
     "@types/node": "^12.11.1",
+    "@typescript-eslint/eslint-plugin": "4.3.0",
+    "@typescript-eslint/parser": "4.3.0",
     "bootstrap": "^4.4.1",
     "codelyzer": "^5.1.2",
+    "eslint": "^7.6.0",
```

Comment:
Can the ``tslint``-related dependencies below be removed?

Reply:
Yes, thanks for the hint! Also, `codelyzer` and `tslint-config-prettier` are not needed anymore.